force-app/main/default/lwc/deleteStiker/deleteStiker.js

```diff
@@ -0,0 +1,21 @@
+
+import LightningModal from 'lightning/modal';
+
+export default class DeleteStiker extends LightningModal {
+
+     handleYes(){
```

Comment:
to reduce function quantity you can use one function and handle different cases based on event